src/man/sssd-simple.5.xml

```diff
@@ -78,8 +80,17 @@
                     <term>simple_allow_users (string)</term>
                     <listitem>
                         <para>
-                            Comma separated list of users who are allowed to
-                            log in.
+                            Comma-separated list of users who are allowed to
+                            log in. If this option is specified, all other
+                            users are denied unless they are listed in
+                            `simple_allow_groups`.
+                        </para>
+                        <para>
+                            **Important:** Defining multiple instances of
+                            `simple_allow_users` will result in only the last
+                            entry being applied. To specify multiple users,
+                            use a **single** `simple_allow_users` entry with a
+                            comma-separated list.
```

Comment:
I think this part should be removed, and also the use of `**` surrounding characters:

`Defining multiple instances of
                            simple_allow_users will result in only the last
                            entry being applied.
`

If you agree, we can keep only

`To specify multiple users,
                            use a **single** `simple_allow_users` entry with a
                            comma-separated list.
`

Reply:
My intention was to call the attention to not use multiple entries with the same attribute, but you have a good point, it is redundant to keep it.